blocks/v2-video/v2-video.css

```diff
@@ -0,0 +1,117 @@
+/* stylelint-disable selector-class-pattern */
+.v2-video-wrapper {
+  --video-inset: inset(32px 48px round 8px);
```

Comment:
This custom property is only used once, on line 89. I expected this on line 17, and its value being overwritten on line 89 and 105.

Reply:
Fixed